core/src/main/java/org/elasticsearch/gateway/TransportNodesListGatewayStartedShards.java

```diff
@@ -191,6 +183,7 @@ public ShardId shardId() {
         public void readFrom(StreamInput in) throws IOException {
             super.readFrom(in);
             shardId = ShardId.readShardId(in);
+            List<DiscoveryNode> discoveryNodes = in.readList(DiscoveryNode::new);
```

Comment:
uhm, this line does not belong here. I wonder why this did not break any tests....